x-pack/legacy/plugins/epm/server/datasources/create.ts

```diff
@@ -90,6 +101,8 @@ async function saveDatasourceReferences(options: {
     datasets,
     assets: toInstall,
   });
+
+  datasource.inputs = inputs;
```

Comment:
@jfsiii I'm adding this here in the hope this also gets stored as part of the datasource which is then available for fleet.

Reply:
@ruflin you need to update `/x-pack/legacy/plugins/ingest/server/mappings.ts` too, but I think we are able to build streams here